cortx-motr.spec.in

```diff
@@ -170,15 +175,45 @@ Conflicts: %{name}
 CORTX Motr unit tests (for kernel and user space).
 %endif # with ut
 
+%if %{with ivt}
+%package ivt
+Summary: CORTX Motr IVT
+Group: Development/Kernel
+Requires: %{name}
+
+%description ivt
+CORTX Motr IVT.
+%endif # with ivt
+
 %prep
 %setup -q
 
 %build
+%if !%{with ivt}
 %configure %{with_linux} %{with_lustre} %{configure_opts}
 make %{?_smp_mflags}
+%else
+%configure %{with_linux} %{with_lustre} %{configure_opts}
+%endif
```

Comment:
```
%configure %{with_linux} %{with_lustre} %{configure_opts}

%if !%{with ivt}
make %{?_smp_mflags}
%endif
```

If `%configure %{with_linux} %{with_lustre} %{configure_opts}` is common, then it can be moved out of the condition.

Reply:
3ce805d